modules/command-bar/src/nextjournal/commands/command_bar.cljs

```diff
@@ -334,7 +336,7 @@
             {:value (bar-state/get-query @!view-state)
              :auto-complete "off"
              :style {:height (if desktop? 20 40) :z-index 3 :width 200 :margin-left 6}
-             :placeholder (if context "Filter…" "⌘J Search commands…")
+             :placeholder (if context "Filter…" (str cmd "J Search commands…"))
```

Comment:
Think it might be better to use `modifier` & `printables` here, see https://github.com/nextjournal/viewers/blob/319e3e42f0a60d4084da416cd2caea4617b0d71b/modules/command-bar/src/nextjournal/commands/keys.cljs#L10-L34

Or we can easily get the `:keys` for the `:toggle` command (which I'm not sure we can) `nextjournal.commands.keys/friendly-path` might be best.